Library/Homebrew/cmd/info.rb

```diff
@@ -86,7 +89,7 @@ def info
     info_args.parse
     # eventually we'll solidify an API, but we'll keep old versions
     # awhile around for compatibility
-    if args.json == "v1"
+    if args.json == "v1" || args.json.present?
```

Comment:
`if args.json == "v1" || args.json == true` ? 

If user passed something else other than `v1` there won't be any warning.